src/firefly/java/edu/caltech/ipac/firefly/server/db/DbAdapter.java

```diff
@@ -0,0 +1,120 @@
+package edu.caltech.ipac.firefly.server.db;
```

Comment:
What are the issues with making `DBAdapter` work with a embedded database? it is out of scope for this ticket but it is worth understanding now.

Reply:
At the moment, DbAdapter is designed to work with embedded database only.  When dealing with remote database, a new SearchProcessor is needed.